Modules/III-k6-Intermediate/02-Dynamic-correlation-in-k6.md

```diff
@@ -4,18 +4,18 @@ In the previous section, you learned [different approaches to debugging a script
 
 ## What is correlation?
 
-Correlation is the process of extracting a value from a previous HTTP response and using that value in the next HTTP request.
+[Data correlation](https://k6.io/docs/misc/glossary/#data-correlation) is the process of extracting a value from a previous HTTP response and using that value in the next HTTP request.
```

Comment:
```suggestion
Data correlation is the process of extracting a value from a previous HTTP response and using that value in the next HTTP request.
```

Reply:
I don't think we need to link to the gloss here since the sentence defines the term.